File: src/Services/API/Domain/Schemas/SchemaMetadata.cs
namespace API.Domain.Schemas
{
    public class SchemaMetadata
    {
        public SchemaDescription Description { get; set; }
        public SchemaOwner Owner { get; set; }
        public string? Script { get; internal set; }
        public IEnumerable<SchemaObject>  ObjectsUsedBySchemaAsync { get; set; } 
    
    }
}

File: src/WebClient/mssql.client/src/app/schema/models/schema-metadata.ts
import { SchemaDescription } from './schema-description';

import { SchemaOwner } from './schema-owner';

import { SchemaObject } from './schema-object';

 
export class SchemaMetadata {
  description: SchemaDescription;
  owner: SchemaOwner;
  script?: string;
  objectsUsedBySchemaAsync: SchemaObject[];
}
